modules/nf-core/bcftools/reheader/tests/main.nf.test

```diff
@@ -0,0 +1,212 @@
+            )
+        }
+
+    }
+
+    test("sarscov2 - [vcf, header, []], []") {
+
+        config "./vcf.config"
+        when {
+
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ],
+                    file(params.test_data['sarscov2']['illumina']['test_vcf_gz'], checkIfExists: true),
+                    file(params.test_data['sarscov2']['illumina']['test_vcf'], checkIfExists: true),
+                    []
+                ]
+                input[1] = [ 
+                    [ id:'genome' ], // meta map
+                    []
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    process.out.vcf,
+                    file(process.out.versions[0]).name,
+                ).match() }
+            )
+        }
+
+    }
+
+    test("sarscov2 - [vcf, [], samples], fai") {
+
+        config "./vcf.config"
+        when {
+
+            process {
+                """
+                ch_no_samples = Channel.of([
+                    [ id:'test', single_end:false ],
+                    file(params.test_data['sarscov2']['illumina']['test_vcf_gz'], checkIfExists: true),
+                    []
+                ])
+                ch_samples = Channel.of(["samples.txt", "new_name"])
+                    .collectFile(newLine:true)
+                input[0] = ch_no_samples.combine(ch_samples)
+                input[1] = [ 
+                    [ id:'genome' ], // meta map
+                    file(params.test_data['sarscov2']['genome']['genome_fasta_fai'], checkIfExists: true)
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    process.out.vcf,
+                    file(process.out.versions[0]).name,
+                ).match() }
+            )
+        }
+
+    }
+
+    test("sarscov2 - [vcf, [], []], fai - stub") {
+
+        options "-stub"
+        config "./vcf.config"
+        when {
+
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ],
+                    file(params.test_data['sarscov2']['illumina']['test_vcf_gz'], checkIfExists: true),
+                    [],
+                    []
+                ]
+                input[1] = [ 
+                    [ id:'genome' ], // meta map
+                    file(params.test_data['sarscov2']['genome']['genome_fasta_fai'], checkIfExists: true)
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    file(process.out.vcf[0][1]).name,
+                    file(process.out.versions[0]).name,
```

Comment:
@mashehu convinced me to snapshot versions

Reply:
So we have to update the snapshots with each release?

Reply:
for now until the next nextflow update

Reply:
or we'll see what we decide on Friday

Reply:
Allright, fine by me!